api.py

```diff
@@ -58,20 +60,21 @@
     console.log("Config must have an email account set up")
     exit(1)
 
-
 def valid_label(label) -> bool:
     # Validates a DNS zone label
     return label and (re.match(r"^(?![0-9]+$)(?!-)[a-zA-Z0-9-]{,63}(?<!-)$", label) is None) and (not label.startswith(".")) and (" " not in label)
 
 
-# Force all projects to have budgets, only needed during initial rollout
-db["projects"].update_many(
-    {"budget": {"$exists": False}},
+# Force all vms to have states, only needed during initial rollout
+db["vms"].update_many(
+    {"state": {"$exists": False}},
     {
-        "$set": {"budget": 2}
+        "$set": {"state": 0}
```

Comment:
Let's run this as a one time job on the database in order to set the initial state. Otherwise all the VMs will reset to 0 every time the API restarts.

Reply:
> Otherwise all the VMs will reset to 0 every time the API restarts.
No I am only doing that where state currently does not exist, but yeah we could move this to a single time job run at the same time as deploy.

Reply:
Ah good point, yes I now see the existence check literally one line above :)

I just ran the update as a one time job, but we can do it again right after merging this to make sure all the VMs are up to date.